docs/stratum.md

```diff
@@ -0,0 +1,123 @@
+AESTRATUM application (alpha)
+==========
+
+Introduction
+==========
+
+By setting of the configuration value `stratum -> enabled` to `true`, the node also starts the aestratum app.
```

Comment:
Can you extend initial paragraph? 

Specify that stratum protocol coordinates pool server and pool clients. Pool server doesn't mine and this is how we deliver it as well (so mining and stratum is exclusive in ae)

Reply:
extended